src/mm-progress-bar/mm-progress-bar.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * @license
+ * Copyright (c) 2015 MediaMath Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at http://mediamath.github.io/strand/LICENSE.txt
+
+*/
+(function(scope) {
+
+	scope.ProgressBar = Polymer({
+		is: 'mm-progress-bar',
+
+		behaviors: [
+			StrandTraits.Stylable
+		],
+
+		properties: {
+			ver: {
+				type: String,
+				value: '<<version>>'
+			},
+			backgroundColor: {
+				type: String,
+				value: '#131313'
+			},
+			fillColor: {
+				type: String,
+				value: '#367ABE'
+			},
+			height: {
+				type: Number,
+				value: 12
+			},
+			indeterminate: {
+				observer: '_updateClass',
+				reflectToAttribute: true,
+				type: Boolean,
+				value: false
+			},
+			percentComplete: {
+				type: Number,
+				value: 0
+			},
+			secondaryFillColor: {
+				type: String,
+				value: '#66ADF2'
+			},
+			width: {
+				type: Number,
+				value: 250
+			}
+		},
+
+		_updateClass: function() {
+			this.fillClass = this.indeterminate ? 'fill fill-indeterminate' : 'fill';
+		},
+
+		updateBarStyle: function(backgroundColor, height, width) {
```

Comment:
given that this and the update method below are essentially exposed as individual properties I think these should both be 'private' (_) and undocumented.